packages/govuk-frontend/src/govuk/components/skip-link/skip-link.mjs

```diff
@@ -31,28 +32,46 @@ export class SkipLink extends GOVUKFrontendComponent {
     this.$module = $module
 
     // Check for linked element
-    const $linkedElement = this.getLinkedElement()
-    if (!$linkedElement) {
-      return
+    try {
+      const $linkedElement = this.getLinkedElement()
+      this.$linkedElement = $linkedElement
+    } catch (cause) {
+      throw new MissingElementError(
```

Comment:
Do we need to document `@throws` in the constructor?

Unlike for `getLinkedElement()` (private method) it'll appear in our JSDoc HTML for this one